server/controllers/article.controller.js

```diff
@@ -166,5 +166,43 @@ class ArticleController {
     }
     return Response(res, 200, 'Articles successfully retrieved', { articles });
   }
+
+  /**
+   * Share article controller
+   * @param {Object} req
+   * @param {Object} res
+   * @returns {Object} response
```

Comment:
The `returns` description in JSDOCS can be more descriptive

Reply:
@geekdanitek Yes, the value returned is actually the response data

Reply:
Ok but it would be nice if there is a description of the return data